pkg/imgutil/qemuimgutil/util.go

```diff
@@ -0,0 +1,87 @@
+// SPDX-FileCopyrightText: Copyright The Lima Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package qemuimgutil
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/lima-vm/lima/pkg/imgutil"
+)
+
+// NewQemuImageUtil returns a new QemuImageUtil instance with "qcow2" as the default format.
+func NewQemuImageUtil() *QemuImageUtil {
```

Comment:
Same as above. Shouldn't this be in `imgutil.go`?
It is hard to distinguish "util" from "imgutil"